dotnet/src/Connectors/Connectors.OpenAI/ToolCallBehavior.cs

```diff
@@ -81,6 +82,11 @@ public static ToolCallBehavior RequireFunction(OpenAIFunction function, bool aut
         return new RequiredFunction(function, autoInvoke);
     }
 
+    /// <summary>
+    /// Options to control function call result serialization behavior.
+    /// </summary>
+    public JsonSerializerOptions? FunctionResultSerializerOptions { get; set; }
```

Comment:
nit: Just double-checking on naming here, because this property is public. Since this is `ToolCallBehaviour` class, should we consider `ToolCallResultSerializerOptions`? I'm wondering if it won't be confusing with our `FunctionResult` type, because it's actual value of `FunctionResult` that is serialized, but not `FunctionResult` class itself.

Reply:
_Since this is ToolCallBehaviour class, should we consider ToolCallResultSerializerOptions?_ I initially thought about it, but then I saw that the ToolCallBehavior class has a function-related member - the AllowAnyRequestedKernelFunction property, and thought that it would be okay to add another function-related property. On the other hand, we probably should avoid adding child class specifics to parent classes. So, the property has been renamed to ToolResultSerializerOptions.